core-web/libs/edit-content/src/lib/interfaces/dot-form.interface.ts

```diff
@@ -0,0 +1,24 @@
+export interface DotForm {
+    row: DotRow;
+}
```

Comment:
I find this confusing, maybe we want an array of `row` instead of an array of `form`. But I'm not sure if that is how it works

Reply:
The final object is like this [ {row: {...}},  {row:....}], was created like this because in the first approach I needed to save an id of the row, with the new version, I think it could be treated directly as [ { columns:[] }, {columns: [] }] and each object can be treatead as a row, what do you think?

Reply:
I think changing it to `[ { columns: [], id: "xxxxxxxxxx" }, { columns: [], id: "xxxxxxxxxx" } ]` is the way to go, like this we have array of rows that have columns. Right now we have an array of forms that have a row